### PR TITLE
Add note about `port` to Mesh GEP

### PR DESCRIPTION
**Pull request by howardjohn**
**What type of PR is this?**
/kind gep

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes-sigs/gateway-api/issues/1487

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -71,6 +71,7 @@ nav:
       - Provisional:
         - geps/gep-1324.md
         - geps/gep-1282.md
+        - geps/gep-1426.md
```
Comment (mikemorris) [inline mkdocs.yml:74]: Fix sort order
```suggestion
        - geps/gep-1426.md
        - geps/gep-1324.md
        - geps/gep-1282.md
```

**Review by mikemorris — APPROVED** [review]
Makes sense to me, and seems to align with how the `port` field on [`ParentReference`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1beta1.ParentReference) is currently used for Gateways.

/approve

**Comment by robscott** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**